web/public/js/services/intercom-service.js

```diff
@@ -8,6 +8,11 @@ angular.module('cpZenPlatform').factory('intercomService', function ($localStora
     return _.contains(user.roles, 'cdf-admin');
   }
 
+  function userIsNinja(user) {
+    var initUserTypeStr = JSON.parse(user.initUserType);
```

Comment:
Ah yes, you're right, but either way, can be initialized only once at the top. lint shouldn't let this go through I think :)